tests/alive-tv/attrs/noundef.srctgt.ll

```diff
@@ -0,0 +1,16 @@
+define i32 @src(i1 noundef %cond) {
```

Comment:
Passing-by question, might be barking at the wrong tree here:
is `{i32, i32} noundef %v` valid? I think alive2 doesn't accept it now.

Reply:
It is valid, but simply unsupported yet. It requires implementation of element-wise undef of inputs.

Reply:
not really. since noundef makes everything non-undef and non-poison. It can be implemented right away I think.

Reply:
IMO padding elements of aggregates is the issue - It can be undef even if noundef is attached.

Reply:
you can still implement it by making all padding undef or not undef, and the remaining values non-undef. Not perfect, but better than nothing.

Reply:
If that is okay, I'll put it in my todo list - I guess the patch wouldn't be very complex to implement.

Reply:
FWIW i asked out of practical interest, i wanted that for https://reviews.llvm.org/D85787 test coverage.